libs/vendored/jwtool/README.md

```diff
@@ -0,0 +1,11 @@
+# jwtool
```

Comment:
This is vendoring in the https://github.com/mozilla/fxa-jwtool, with relatively minimal modernization to TS based classes and async/await. I was trying to avoid having to make further updates through-out the code-base where its used, so I didn't change any of the API method names or naming of this package (except removing 'fxa' since its in the fxa monorepo now).